addons/binding/org.openhab.binding.gpstracker/README.md

```diff
@@ -86,41 +86,59 @@ If the things are not defined in **.things** files the first time the tracker se
 Basic channels provided by the tracker things:
 
 * **Location** - Current location of the tracker
+* **Accuracy** - GPS accuracy
 * **Last Report** - Timestamp of the last location report
 * **Battery Level** - Battery level of the device running the tracker application
 * **Region trigger channel** - Used by regions defined in tracker application. Event is fired with payload of the region name when the binding receives a **transition** log record or a distance calculation for a **location** record indicates that the tracker is outside of the region circle. Payload is suffixed with `/enter` for entering and with `/leave` for leaving events.
 
 #### Distance Calculation
 
-Tracker thing can be extended with **Distance** channels if a distance calculation is needed for a region. 
+Tracker thing can be extended with **Distance** channels (channel type is `regionDistance`) if a distance calculation is needed for a region. 
 These dynamic channels require the following parameters:
 
 | Parameter | Type |Description |
 | --- | --- | --- |
 | Region Name | String | Region name. If the region is configured in the tracker app as well use the same name. Distance channels can also be defined as binding only regions (not configured in trackers) | 
 | Region center | Location | Region center location |
+| Region Radius | Integer | Geofence radius |
```

Comment:
Please apply the Markdown Table formatter: https://www.tablesgenerator.com/markdown_tables

Reply:
Ok